proto/stride/stakeibc/tx.proto

```diff
@@ -314,3 +316,27 @@ message MsgUpdateTradeRoute {
   ];
 }
 message MsgUpdateTradeRouteResponse {}
+
+// Registers or updates a community pool rebate by specifying the amount liquid
+// staked
+message MsgSetCommunityPoolRebate {
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stride/x/stakeibc/MsgSetCommunityPoolRebate";
+
+  // Message signer (admin only)
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+  // Chain id of the chain whose community pool has a liquid staking rebate
+  // arrangement with stride
+  string chain_id = 2;
+  // Rebate percentage (e.g. 0.5 for 50%)
```

Comment:
nit: can we make this `e.g. 0.2 for 20% rebate, Stride passes 20% of fee on `liquid_staked_amount` to community pool`? Just to remove ambiguity!

Reply:
yes good call! done